### ISSUE TITLE
Make the kernel work with all grammars in the language-stata package

### ISSUE THREAD

**Issue by sikaiser** [kylebarron/stata_kernel#186]
**Is your feature request related to a problem? Please describe.**
Not a problem.

**Describe the solution you'd like**
When writing a dyndoc file, it is handy to be able to test parts of the code using the kernel. (Personally I even like to put the line that executes the whole script into the script itself.) Currently the only way to do this is to manually switch the language to "Stata", execute the code, then switch the language back. It would be nice if it would work out of the box for all the grammars.

**Describe alternatives you've considered**
Leaving it as is and manually switching back and forth.

**Comment by bermudaparty** [comment]
Unfortunately none of these work for me. I see that `source.stata` itself is also not listed there?

**Comment by kylebarron** [comment]
These are grammars for files that have languages embedded within them. Like `text.md`.

**Comment by kylebarron** [comment]
If you add `"stata": "Stata Dyndoc (Markdown)"` to the Language Mappings in the settings it'll work.

**Comment by kylebarron** [comment]
In conjunction with adding `source.dyndoc.md.stata` to the list.

**Comment by bermudaparty** [comment]
I did some testing.

Basically, adding the mapping to Hydrogen preferences is sufficient. This works for all three `Dyndoc ` languages without adding anything to the hydrogen `utils.js`.

The problem arises when adding the basic `Dyndoc` language AND either `Dyndoc (Markdown)` or `Dyndoc (LaTeX)`. In that case the language that is second in the list still gets recognized, but not the first one. (I tested this because I'm assuming it might have to do with how the 'sub'-languages `include source.dyndoc.stata`. But if I reverse the order, it also changes which one gets recognized and which not.)

Combining `Dyndoc (Markdown)` and `Dyndoc (LaTeX)` works fine on the other hand.

So I guess this is still a Hydrogen issue, but a different one?
